core/App/ehsm_core_test.cpp

```diff
@@ -409,6 +409,41 @@ void test_export_datakey()
     printf("============test_export_datakey end==========\n");
 }
 
+void test_GenerateQuote_and_VerifyQuote()
```

Comment:
change the commit title as 
"
add a fake implementation for remote attestation apis
"